CoseSignTool/ValidateCommand.cs

```diff
@@ -133,7 +133,7 @@ public override ExitCode Run()
             Console.Error.WriteLine(result.ToString());
 
             return result.Success ? ExitCode.Success
-                : result.Errors?.Count > 0 ? ErrorMap[result.Errors.FirstOrDefault().ErrorCode]
+                : result.Errors?.Count > 0 || result.Errors is not null ? ErrorMap[result.Errors.FirstOrDefault().ErrorCode]

```

Comment:
When are we getting a non-null but empty result.Errors collection? Shouldn't there be an error in the collection if validation failed?

Reply:
Yeah I think you're right about that!